vitest.config.ts

```diff
@@ -5,5 +5,6 @@ export default defineConfig({
         environment: 'jsdom',
         globals: true,
         setupFiles: ['./vitest.setup.ts'],
+        watch: false,
```

Comment:
Høres mer ut som ett lokalt problem, du kan øke dette på egen maskin, ville hatt standard til å ha `watch` på for utvikling.